pkg/aks/components/AksNodePool.vue

```diff
@@ -0,0 +1,317 @@
+      modeOptions:             ['User', 'System'] as AKSPoolMode[],
+      availabilityZoneOptions: [{ label: 'zone 1', value: '1' }, { label: 'zone 2', value: '2' }, { label: 'zone 3', value: '3' }],
+    };
+  },
+
+  watch: {
+    'pool.enableAutoScaling'(neu) {
+      if (!neu) {
+        delete this.pool.minCount;
+        delete this.pool.maxCount;
+      } else {
+        this.$set(this.pool, 'minCount', 1);
+        this.$set(this.pool, 'maxCount', 3);
+      }
+    },
+    'pool.vmSize'(neu) {
+      if (neu) {
+        this.$emit('vmSizeSet');
+      }
+    }
+  },
+
+  methods: {
+    addTaint() {
+      this.taints.push({ taint: '', _id: randomStr() });
+      this.$set(this.pool, 'nodeTaints', this.taints.map((keyedTaint: any) => keyedTaint.taint));
+      this.$emit('input');
+    },
+
+    updateTaint(keyedTaint: any, idx: any) {
+      this.taints[idx] = keyedTaint;
+      this.$set(this.pool, 'nodeTaints', this.taints.map((keyedTaint: any) => keyedTaint.taint));
+      this.$emit('input');
+    },
+
+    removeTaint(idx: number) {
+      this.taints.splice(idx, 1);
+    }
+  },
+});
+</script>
+
+<template>
+  <div class="pool">
+    <div class="row mb-10">
+      <div class="col span-3">
+        <LabeledInput
+          v-model="pool.name"
+          :mode="mode"
+          label-key="generic.name"
+          required
+          :disabled="!pool._isNewOrUnprovisioned"
+        />
+      </div>
+      <div class="col span-3">
+        <LabeledSelect
+          v-model="pool.vmSize"
+          :options="vmSizeOptions"
+          label-key="aks.nodePools.vmSize.label"
+          :loading="loadingVmSizes"
+          :mode="mode"
+          :disabled="!pool._isNewOrUnprovisioned"
+          :rules="[()=>pool._validSize === false ? 'This size is not avaiable in the selected region' : undefined]"
+        />
+      </div>
+      <div class="col span-3">
+        <LabeledSelect
+          v-model="pool.availabilityZones"
+          :options="availabilityZoneOptions"
+          label-key="aks.nodePools.availabilityZones.label"
+          :mode="mode"
+          :multiple="true"
+          :taggable="true"
+          :disabled="!pool._isNewOrUnprovisioned"
+        />
+      </div>
+      <div class="col span-2">
+        <RadioGroup
+          v-model="pool.mode"
+          :mode="mode"
+          :options="modeOptions"
+          :name="`${pool._id}-mode`"
+          :disabled="!pool._isNewOrUnprovisioned"
+          :row="true"
+          label-key="generic.mode"
+          @input="$emit('validationChanged')"
+        >
+          <template #label>
+            <span class="text-label">Mode</span>
+          </template>
+        </RadioGroup>
+      </div>
+    </div>
+
+    <div class="row mb-10">
+      <div class="col span-3">
+        <LabeledSelect
+          v-model="pool.osType"
+          :options="[]"
+          label-key="aks.nodePools.osType.label"
+          mode="view"
```

Comment:
Thanks for reminding me, this is always disabled in Ember and I forgot to go back and investigate why...From what I can tell going off git history, (sparse) github comments, and QA test plan confluence history, AKS windows support was initially planned then scrapped. This input was not editable in any Rancher release. 

I've removed the input entirely. I'm not sure if we need to explicitly stated that the nodes created here are going to be Linux nodes? My understanding is that windows k8s nodes are somewhat...niche and less supported. But I could be convinced otherwise.